### PR TITLE
Create the page header component is used for NFTs screens

### PR DESCRIPTION
**Pull request by bulatsalyakhovdev**
[Freedcamp 40212219](https://freedcamp.com/view/2956183/tasks/40212219)

image

### CODE REVIEW

**Comment by dbizhev** [comment]
I think @CrazyOptimist is a perfect reviewer for this PR.

File: src/common/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export interface PageHeaderProps {
+  title: string
+  routes: PageHeaderRoutes[]
+  sortByOptions: PageHeaderSortByOptions[]
+}
+
+/**
+ * This is a presentational component, meaning it holds no state.
+ * State should be handled by parent component.
+ */
```
Comment (CrazyOptimist) [inline src/common/components/PageHeader/PageHeader.tsx:27]: We know it by reading the code, it's self explaining.

File: src/common/components/PageHeader/PageHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import Select, { Option } from '../Select/Select'
+import cn from 'classnames'
+
+export type PageHeaderRoutes = {
+  label: string
+  route?: string // in the future when clicking the link it should change current route...
```
Comment (CrazyOptimist) [inline src/common/components/PageHeader/PageHeader.tsx:7]: we are good to remove this comment. :)

Comment (bulatsalyakhovdev) [reply src/common/components/PageHeader/PageHeader.tsx:7]: @CrazyOptimist we are all team, we work together, so good comments would be helpful for everyone to understand codes.

**Pull request merged**